packages/gatsby/src/joi-schemas/joi.js

```diff
@@ -40,6 +40,7 @@ export const nodeSchema = Joi.object()
         fieldOwners: Joi.array(),
         content: Joi.string().allow(``),
         description: Joi.string(),
+        internal: Joi.boolean(),
```

Comment:
should this be `ignoreType` ?

Reply:
:face_with_head_bandage: it should. Thanks.